ppr-ui/src/views/mhrSearch/ConfirmMHRSearch.vue

```diff
@@ -269,15 +269,15 @@ export default class ConfirmMHRSearch extends Vue {
     if (this.isRoleStaffReg) {
       apiResponse = await submitSelectedMhr(
         this.getManufacturedHomeSearchResults.searchId,
-        this.getSelectedManufacturedHomes,
+        uniqBy(this.getSelectedManufacturedHomes, UIMHRSearchTypeValues.MHRMHR_NUMBER),
         this.getFolioOrReferenceNumber,
         this.getStaffPayment,
         this.isSearchCertified
       )
     } else {
       apiResponse = await submitSelectedMhr(
         this.getManufacturedHomeSearchResults.searchId,
-        this.getSelectedManufacturedHomes,
+        uniqBy(this.getSelectedManufacturedHomes, UIMHRSearchTypeValues.MHRMHR_NUMBER),
```

Comment:
So to clarify this will squash all instances of searches where mhr number is the same?
Is there any of the search criteria categories where this would be unwanted?

Reply:
Thats correct. Basically doing what the UI does already but when passing to the API. All MHR search categories function in this way.

Reply:
Ok thank you for confirming 👍